src/azure-cli-core/azure/cli/core/test_utils/vcr_test_base.py

```diff
@@ -46,6 +46,10 @@
 
 # MOCK METHODS
 
+# Workaround until https://github.com/kevin1024/vcrpy/issues/293 is fixed.
+vcr_connection_request = vcr.stubs.VCRConnection.request
+vcr.stubs.VCRConnection.request = lambda *args, **kwargs: vcr_connection_request(*args)
```

Comment:
I'm pretty sure flake8 will bad mouth this line of code. But we disable the style check here anyway.